openshift-knative-operator/pkg/serving/secretinformerfiltering.go

```diff
@@ -74,15 +70,10 @@ func configIfUnsetAndCheckIfShouldInject(comp *operatorv1beta1.KnativeServing, d
 		}
 	}
 
-	// The annotation is deprecated
-	// TODO: Remove this block in future releases
-	if deployment == "net-istio-controller" {
-		if v, ok := comp.GetAnnotations()[secretInformerFilteringAnnotation]; ok {
-			b, _ := strconv.ParseBool(v)
-			// Keep the same behavior as in 1.27
-			return b, common.InjectEnvironmentIntoDeployment("net-istio-controller", "controller",
-				corev1.EnvVar{Name: EnableSecretInformerFilteringByCertUIDEnv, Value: v})
-		}
+	// TODO: remove when set to true at the net-* repos upstream/midstream
```

Comment:
Same here, JIRA?

Reply:
See https://issues.redhat.com/browse/SRVKS-1190 and https://issues.redhat.com/browse/SRVKS-1191.